SCDoc/SCDoc.y

```diff
@@ -200,8 +200,8 @@ subsubsection: METHOD methnames optMETHODARGS eol methodbody
 
                 $$ = doc_node_make(
                 stringEqual(method_type, "CMETHOD") ? "CCOPYMETHOD"
-                                                    : (stringEqual(method_type, "IMETHOD") ? "ICOPYMETHOD"
-                                                                                           : "COPYMETHOD"),
+                                                   : (stringEqual(method_type, "IMETHOD") ? "ICOPYMETHOD"
+                                                                                          : "COPYMETHOD"),
```

Comment:
you deleted 2 whitespaces
```suggestion
                                                    : (stringEqual(method_type, "IMETHOD") ? "ICOPYMETHOD"
                                                                                           : "COPYMETHOD"),
```

We can either rebase this again, maybe easier, just apply the suggestion and squash the commits afterwards so they don't show up in the history.